beacon-chain/node/node.go

```diff
@@ -91,6 +92,12 @@ func NewBeaconNode(cliCtx *cli.Context) (*BeaconNode, error) {
 		return nil, err
 	}
 
+	// Warn if user's platform is not supported
+	err := prereq.WarnIfNotSupported(cliCtx.Context)
+	if err != nil {
+		return nil, err
```

Comment:
We should not stop execution on error (please follow the comments in `WarnIfNotSupported` func). Warn if not supported, should just warn (if system version is not detectable, no error, warn that system version cannot be determined).

Reply:
Thanks. Will address in upcoming commit